.test-infra/kubernetes/kafka-cluster/05-kafka/job-config.yaml

```diff
@@ -0,0 +1,40 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+---
+# Source: kafka/templates/job-config.yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "kafka-config-eff079ec"
+  namespace: kafka
+spec:
+  template:
+    metadata:
+      labels:
+        app: kafka
+    spec:
+      restartPolicy: OnFailure
+      volumes:
+        - name: config-volume
+          configMap:
+            name: kafka-config
+            defaultMode: 0744
+      containers:
+        - name: kafka-config
+          image: "confluentinc/cp-kafka:5.0.1"
+          command: ["/usr/local/script/runtimeConfig.sh"]
+          volumeMounts:
+            - name: config-volume
+              mountPath: "/usr/local/script"
```

Comment:
no newline - please add

please apply this to any other files that do not end with newlines

Reply:
Sure